tests/test_flatten.py

```diff
@@ -15,20 +13,18 @@
 GEOPD_DATA = GeoDataFrame(DICT_DATA, crs="EPSG:4326", geometry=geometry)
 
 
-
-
 def teste_flatten_wrong_nested_columns_value_error():
     with raises(ValueError):
-        flatten(DICT_DATA, nested_columns="wrong")
+        flatten(DICT_DATA, nested_columns={"wrong"})
```

Comment:
Just realized that I could have passed `nested_columns` as list...

Reply:
If you [say](https://github.com/FelipeSBarros/crossfire/pull/100#discussion_r1487082934) that a regular user might pass a list, I think tests would reflect that indeed…